SwiftGit2Tests/Fixtures/Fixtures.swift

```diff
@@ -47,12 +53,20 @@ final class Fixtures {
 	}
 	
 	func unzipFileAtURL(fileURL: NSURL, intoDirectoryAtURL directoryURL: NSURL) {
+#if os(OSX)
+
 		let task = NSTask()
 		task.launchPath = "/usr/bin/unzip"
 		task.arguments = [ "-qq", "-d", directoryURL.path!, fileURL.path! ]
-		
+            
 		task.launch()
 		task.waitUntilExit()
+
+#else
+
+		assertionFailure("Tests not supported on iOS yet")
```

Comment:
I'm :+1: to adding that in a separate PR. Let's just add a comment here then explaining that the fixtures can't be unzipped.